packages/web-config-server/src/apiV1/dataBuilders/generic/table/tableOfDataValues/tableOfPercentagesOfValueCounts.js

```diff
@@ -3,18 +3,16 @@
  * Copyright (c) 2020 Beyond Essential Systems Pty Ltd
  */
 
-import flatten from 'lodash.flatten';
 import { getPercentageCountOfValuesByCell } from './getValuesByCell';
 
 import { TableOfDataValuesBuilder } from './tableOfDataValues';
 
 class TableOfPercentagesOfValueCounts extends TableOfDataValuesBuilder {
   buildDataElementCodes() {
-    const dataElementCodes = flatten(
-      flatten(this.config.cells).map(cell =>
-        cell.numerator.dataValues.concat(cell.denominator.dataValues),
-      ),
-    );
+    const dataElementCodes = this.config.cells
+      .flat()
```

Comment:
Hooray!

Reply:
Yay! 🎉 I created [1240: Use modern ES syntax](https://github.com/beyondessential/tupaia-backlog/issues/1240) to do the same in the rest of the codebase